db/migrate/20231025130520_create_post.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePost < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.integer :author_id
+      t.string :title
+      t.string :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :user, foreign_key: true
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_2.md#:~:text=should%20be%20included.-,Pay%20attention%20that%20in%20posts%20table%20the%20column%20holding%20the%20foreign%20key%20to%20the%20users%20table%20should%20be%20named%20author_id,-All%20columns%20that), The foreign key between `posts` and `users` tables should be `author_id`, not `user_id` as you did. Please fix that.

[Hint] This how the reference should be:
```ruby
 t.references :author, foreign_key: { to_table: 'users' }
```